### PR TITLE
Feature/activity chart

### PR DESCRIPTION
**Pull request by MayuRuru**

### CODE REVIEW

File: test/test_utils/TestMaterialAppWidget.dart [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+        //title: 'Widget Test',
+        home: home,
+        //navigatorObservers: [navigatorObserver],
+        //routes: _testRoutes(),
+      ),
+    );
+  }
+}
+
+/*  Map<String, WidgetBuilder> _testRoutes() {
```
Comment (selankon) [inline test/test_utils/TestMaterialAppWidget.dart]: Commented code MA-LO

File: .gitlab-ci.yml [hunk 2]
```diff
@@ -13,7 +13,7 @@ build-android-apk:
   stage: build
   only:
     - develop
-    - master
+    - main
```
Comment (selankon) [inline .gitlab-ci.yml]: This is not the place, should be on a separated PR

File: android/app/src/main/AndroidManifest.xml [hunk 3]
```diff
@@ -1,6 +1,9 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="com.example.empriusapp">
     <uses-permission android:name="android.permission.INTERNET"/>
+    <uses-permission android:name="android.permission.CAMERA" />
+    <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" />
+    <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" />
```
Comment (selankon) [inline android/app/src/main/AndroidManifest.xml]: Same, this have to be on a separated PR

**Review by selankon — COMMENTED** [review]
Very good job!! 

You have to try to separate better the code on different PR's. This, for example, has 53 changes which causing difficulty to see the relevant code changes. 

Also for the mixed stuff: added a gitlabci, change android permissions, are not related to the activity chart.

But for this time no worries, you are doing a good job. 

IMHO is ready to merge if there are no merge conflicts!

**Comment by selankon** [comment]
Seems that there are conflicts, u should rebase with main probably

File: test/test_utils/TestMaterialAppWidget.dart (hunk 1 above)
Comment (MayuRuru) [reply test/test_utils/TestMaterialAppWidget.dart]: tests should not be here its a WIP branch indeed

File: .gitlab-ci.yml (hunk 2 above)
Comment (MayuRuru) [reply .gitlab-ci.yml]: sure

**Pull request merged**